package.json

```diff
@@ -88,5 +92,15 @@
       "src/**/*.js"
     ],
     "babel": "inherit"
+  },
+  "lint-staged": {
+    "src/**/*.js": [
+      "prettier --trailing-comma all --write ",
+      "git add"
+    ],
+    "test/**/*.test.js": [
```

Comment:
@hzoo I think the only reason why can't do this is that running on all .js files will affect fixtures